src/components/reporter/findings/fields.js

```diff
@@ -1,14 +1,14 @@
 export const emailField = {
   name: "email",
-  label: "Email address",
+  label: "Email address *",
```

Comment:
Is it possible to accomplish this a different way? CSS + required?

```suggestion
  label: "Email address",
```

Reply:
Good idea! I added it to the `Input` component and left a comment in this file to delete them once we migrate to using the `Input` everywhere.